perfmetrics/scripts/gcsfuse_goofys_fio_test.py

```diff
@@ -0,0 +1,143 @@
+"""Python script to do FIO test for GCSFuse Vs GCSFuse/Goofys.
+
+To run the script:
+>> python3 gcsfuse_goofys_fio_test.py -- method1 method1_version method2 method2_version jobfile_path
```

Comment:
call the arguments as fuse_type_1, fuse_type_2, fuse_type_1_version, fuse_type_2_version